### PR TITLE
-msse3 clang flag

### PR DESCRIPTION
**Pull request by debris**
#661

### CODE REVIEW

File: cmake/EthCompilerSettings.cmake [hunk 1]
```diff
@@ -25,8 +25,10 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
 	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -DETH_DEBUG")
 	set(ETH_SHARED 1)
 
+	# For macos we need to set CLANG_X86_VECTOR_INSTRUCTIONS to ssse3
+	# This should result in binaries which will execute on Apple hardware which are < Mavericks
 	if ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux")
-		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libstdc++ -fcolor-diagnostics -Qunused-arguments -DBOOST_ASIO_HAS_CLANG_LIBCXX")
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libstdc++ -fcolor-diagnostics -Qunused-arguments -DBOOST_ASIO_HAS_CLANG_LIBCXX -msse3")
```
Comment (gavofyork) [inline cmake/EthCompilerSettings.cmake]: this is the linux condition - shouldn't this be under an apple/darwin condition?

Comment (subtly) [inline cmake/EthCompilerSettings.cmake]: This doesn't fix this issue. Also worth noting, I think CMAKE_SYSTEM_NAME is Linux even when it's macos.

**Comment by subtly** [comment]
@debris This won't fix #661. I know for sure that setting CLANG_X86_VECTOR_INSTRUCTIONS to ssse3 works; not sure what -msse3 does. As noted; this should only apply for macos builds.

**Comment by debris** [comment]
After discussing issue with @subtly, I'm closing the pr. #661 needs to be updated.

**Pull request closed without merging**